packages/blockchain-wallet-v4-frontend/src/components/Alerts/template.tsx

```diff
@@ -1,14 +1,12 @@
 import React from 'react'
-import ReactCSSTransitionGroup from 'react-addons-css-transition-group'
+import { AnimatePresence,motion } from 'framer-motion'
```

Comment:
space is missing somehow should be `{ AnimatePresence, motion } `

Reply:
How did this happen?